docs/modules/operation/pages/deep-dive/notifications/concepts.adoc

```diff
@@ -94,3 +94,20 @@ This switch specifies the logic used when deciding whether to send a notice for
 If the switch is `off`, notices will never be sent to the target.
 If the switch is `on`, notices will always be sent to the target.
 If the switch is set to `auto`, the system deploys heuristics aimed at "doing the right thing."
+
+== Variable bindings
```

Comment:
This is a perfect use case for this functionality: https://chat.opennms.com/opennms/pl/gnyui5sgwiddpfi1moymrrj8ry

Reply:
Thanks for this feedback.  Honestly, I've never used either of those settings on notifications, so this doc update is welcomed.

Reply:
According to the XSD, it is 1 varbind and many parameters
```
        <element maxOccurs="unbounded" minOccurs="0" ref="this:parameter"/>
        <element maxOccurs="1" minOccurs="0" ref="this:varbind"/>
```